portfolio/src/main/java/com/google/sps/servlets/Server2.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.servlets;
+
+
+import java.io.IOException;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/head")
+public class Server2 extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    ArrayList<String> temp = new ArrayList<String>();
+    temp.add("This is cool");
+    temp.add("I am confused");
+    temp.add("I don't know if I can do this alone");
```

Comment:
How are you doing?

Reply:
I'm doing well. I went to office hours and was able to complete it. I just used those as an example because that was what I was going through. Thanks for checking up on me :)

Reply:
Good, I'm very glad thing are looking up!